_posts/1100-09-01-AWS-installing-on-multiple-accounts.md

```diff
@@ -0,0 +1,40 @@
+---
+date: 2018-04-17
+title: Multiple AWS Accounts
+categories:
+    - amazon web service
+description: "A guide for installing Spinnaker on multiple AWS accounts from a single repo"
+type: Document
+---
+Below we will explain how switching between environment profiles, gives the ability to switch between AWS account when you have a single Armory Spinnaker installation.
+
+## What you will need
+
+A file similar to */opt/spinnaker/env/prod.env* The name *prod* comes from the *stack* in a servergroup.
```

Comment:
`/opt/spinnaker/env/prod.env` is the path on the box. If these directions are meant for people using a config repo similar to `armory/spinnaker-config-deg` then we should give the relative path from the root of the repo. People won't necessarily know about how things get put on the box.

Reply:
Should I use {relative path}/prod.env?